game_api.txt

```diff
@@ -473,18 +473,18 @@ The player API can register player models and update the player's appearance.
 ### Model Definition
 
 	{
-		animation_speed = 30,           -- Default animation speed, in FPS
-		textures = {"character.png", }, -- Default array of textures
-		visual_size = {x = 1, y = 1},   -- Used to scale the model
+		animation_speed = 30,           -- Default animation speed, in keyframes per second
+		textures = {"character.png"},   -- Default array of textures
 		animations = {
-			-- <anim_name> = {x = <start_frame>, y = <end_frame>},
-			foo = {x = 0, y = 19},
-			bar = {x = 20, y = 39},
-			-- ...
+			-- [anim_name] = {x = <start_frame>, y = <end_frame>, collisionbox = default collisionbox, eye_height = default eye height},
```

Comment:
I do not understand why these are documented as 'default collisionbox' and 'default eye height', surely these are not the defaults. How about:
`collisionbox = <anim collisionbox>, eye_height = <anim eye height>`

Reply:
`<anim_name>`
seems more suitable for 'insert your value here', as in previous code.

Reply:
Hmm, my thought was that if you do `key = <what_the_value_should_be>` it's not the same as `key = default value`. I wanted to make clear that there are defaults, but this is probably misleading.